test/JDBC/expected/BABEL-5103-vu-prepare.out

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE BABEL_5103_T1(
```

Comment:
Lets add meaningful name of this testfile.

Reply:
Renamed file to BABEL-CASE_EXPR. 
Also added Null tests